AllReadyApp/Web-App/AllReady/Areas/Admin/Controllers/TaskController.cs

```diff
@@ -55,6 +58,15 @@ public IActionResult Create(int activityId, TaskEditModel model)
                 ModelState.AddModelError("EndDateTime", "Ending time cannot be earlier than the starting time");
             }
 
+            var activity = _dataAccess.GetActivity(activityId);
+            if ((model.StartDateTime < activity.StartDateTime || model.EndDateTime > activity.EndDateTime) && model.IgnoreWarning == false)

```

Comment:
For tasks, model.StartDateTime and model.EndDateTime can null. In that case I don't think we want this validation warning to show.